### ISSUE TITLE
Cannot get recording working

### ISSUE THREAD

**Issue by Ace71425** [alexismorin/OpenMocap#3]
I'm in Unity 2019.2.14f1 .. I did follow the steps of removing the "direct to camera" setting and adding OpenVR....everything appears to be working as far as the scene goes but hitting the trigger does not record anything, thus I tried to use recorder manually and the anim files that it creates don't see to do anything except twitch the model around without actually doing any hand or body movements. Any ideas?

**Comment by ChunkyToads** [comment]
@Ace71425 had similar issue myself trying this out... Honestly, bought a kinect v2 and used nimate w/ blender. The kinect works great for general body movements and excellent for facial mocap with lipsync (if you pay for brekel face app). Ultimately though could get just as much done body wise with AutoRigPro now that is a worthwhile purchase for sure!

**Comment by Ace71425** [comment]
Yeah ive been at this for years and I was recently rejuvenated by seeing this project..my thought was that if I can use an open source add on in unity like this and combine it with kinect to VR software to combine the power of the rift and the kinect and then contribute to the project and tweak the code myself i could have a pretty accurate mocap attachment to every unity project...but i cant even get this thing to record animations let alone see if it can accept Kinect sensors in steamVR and then further develop the prediction calculations

**Comment by ChunkyToads** [comment]
@Ace71425 Honestly, OpenMocap is cool but seems to essentially be just VR tracking hooked up with Inverse Kinematics and some associated leg movement animations.  I've seen nearly the exact thing accomplished for Oculus devices on youtube, I think the user 'Valem' put something out in that regard. I also recently used an opensource project with SteamVR that works very well on the Index or Vive (no additional trackers needed either) https://github.com/WireWhiz/VR-Instincts.  Going back to non-real-time, the kinect's limitation on recording mostly forward facing movement is easily overcome by using it to just record the natural jitter most people have when moving or standing, and  transferring the info to a AutoRigPro armature. I usually do that and then add on any 360 limb or body rotations ontop the end result is indistinguishable IMO. Still, sounds like you're looking for more of an in-engine real-time solution. If you ever end up going the non-real-time route and would like any tips n tricks in blender for the setup hit me up (ChunkyToads#8868 on discord).  I've kept thorough notes and have utilized it regularly on a variety of models. For now good luck.

**Comment by Ace71425** [comment]
Yeah I'm not married to any specific idea I just have a kinect and a rift
and would love to find a good solid solution for motion capture...a blender
solution would be great too...i sent you a discord friend request would be
interested to hear about your setup
